api/seeds/staging.js

```diff
@@ -0,0 +1,11 @@
+const roles = require('./shared/roles-and-activities');
+
+exports.seed = async knex => {
+  // Don't seed this data if we're not in a development environment.
+  if (process.env.NODE_ENV !== 'staging') {
+    return;
+  }
+
+  // Call specific seeds from here.
+  await roles.seed(knex);
```

Comment:
For staging, we don't want to delete anything - user accounts that were created, form data that was submitted, etc.  Instead, just update the roles and activities to add anything new that we've created.

Reply:
niiice